pkg/spec/tf_job.go

```diff
@@ -107,6 +107,56 @@ func (c *TfJobSpec) Validate() error {
 	return nil
 }
 
+// ConfigureAccelerators adds any accelerator specific configuration to the pods.
+func (c *TfJobSpec) ConfigureAccelerators(accelerators map[string]AcceleratorConfig) error {
+	for _, r := range c.ReplicaSpecs {
+		if r.Template == nil {
+			return fmt.Errorf("Replica is missing Template; %v", util.Pformat(r))
+		}
+		for i, c := range r.Template.Spec.Containers {
+			if c.Name == string(TENSORFLOW) {
```

Comment:
I wanted to allow for more than 1 container in the pod. The original reason for that was to handle logging via a sidecar container. If we don't need to allow for more than 1 container than we can drop this requirement. Alternatively, we could introduce some other method to identify the container that is running TensorFlow.